main.py

```diff
@@ -1,7 +1,21 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
-
+from db.models import Actor, Genre
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
There is a typo in the actor's name 'George Klooney'. It should be 'George Clooney'. This is corrected later in the code, so it might be intentional for demonstration purposes.